viewing_party/party.py

```diff
@@ -0,0 +1,141 @@
+## TEST WAVE 1 ##
+
+def create_movie(movie_title, genre, rating):
+    movie = {}
+    if movie_title and genre and rating:
+        movie["title"] = movie_title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie
+    return None
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+def watch_movie(user_data, movie_title):
+    movie_transfer = {}
+    for list_index in user_data["watchlist"]:
+        if movie_title in list_index.values():
+            movie_transfer.update(list_index)
+            user_data["watchlist"].pop()
+            user_data["watched"].append(movie_transfer)
+    return user_data
+
+
+def get_watched_avg_rating(user_data):
+    rating_list = []
+    if len(user_data["watched"]) > 0:
+        for watched_key in user_data["watched"]:
+            rating_list.append(watched_key["rating"])
```

Comment:
This is a whole dictionary, not a key:

```suggestion
        for movie in user_data["watched"]:
            rating_list.append(movie["rating"])
```